.github/workflows/repo2docker-workflow.yaml

```diff
@@ -0,0 +1,28 @@
+name: Workflow to push the latest built docker image to dockerhub
+on:
+  push:
+    branches: [ master+ipynb ]
+
+jobs:
+  binder:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Code
+      uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+        
+    - name: Build scripts
+      run: |
+        wget https://nightly.link/OpenMS/OpenMS/workflows/pyopenms-wheels/nightly/Linux-wheels.zip\?status\=completed
+        mv Linux-wheels.zip\?status=completed Linux-wheels.zip
+        unzip Linux-wheels.zip
+        pip install pyopenms_nightly-*-cp37-cp37m-manylinux2014_x86_64.whl
+
+    - name: update jupyter dependencies with repo2docker
+      uses: jupyterhub/repo2docker-action@master
+      with:
+        DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+        DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
```

Comment:
@timosachsenberg can you please add the docker credentials in the secret section?